components/count-badge/count-badge.js

```diff
@@ -48,14 +50,45 @@ class CountBadge extends RtlMixin(LitElement) {
 			 */
 			hideZero: {
 				type: Boolean,
-				reflect: true,
 				attribute: 'hide-zero'
 			},
+			/**
+			 * The text that will show as an aria-label on the badge. NOTE: Only the text will be read by screen-readers (not the number).
+			 * @type {string}
+			 */
+			text: {
+				type: String,
+				attribute: 'text'
+			},
+			/**
+			 * Optionally choose to add a tab stop to the badge. Defaults to false.
+			 * @type {boolean}
+			 */
+			tabStop: {
+				type: Boolean,
+				attribute: 'tab-stop'
+			},
+			/**
+			 * Optionally choose to announce changes to the badge with an aria-live region. If the number property is changed, the text will be read by screenreaders. Defaults to false.
+			 * @type {boolean}
+			 */
+			announceChanges: {
+				type: Boolean,
+				attribute: 'announce-changes'
+			},
+			/**
+			 * Optionally add a tooltip on the badge. Defaults to false.
+			 * @type {boolean}
+			 */
+			tooltip: {
```

Comment:
Nit: could this be `has-tooltip`? Just want the name to clearly be "boolean" as we use `tooltip` on `<d2l-switch>` as a String.